client/src/Try/Gist.js

```diff
@@ -1,48 +1,10 @@
 "use strict";
 
-exports.getGistById_ = function(id, done, fail) {
-  $.ajax({
-    url: 'https://api.github.com/gists/' + id,
-    dataType: 'json'
-  }).done(done).fail(function(err) {
-    fail("Unable to load Gist metadata");
-  });
-}
-
-exports.tryLoadFileFromGist_ = function(gistInfo, filename, done, fail) {
-  if (gistInfo.files && gistInfo.files.hasOwnProperty(filename)) {
-    var url = gistInfo.files[filename].raw_url;
-
-    return $.ajax({
-      url: url,
-      dataType: 'text'
-    }).done(done).fail(function(err) {
-      fail(err.statusText);
-    });
-  } else {
-    fail("Gist does not contain a file named " + filename);
+exports.rawUrl_ = function (gistInfo_, filename) {
+  try {
+    var gistInfo = JSON.parse(gistInfo_);
+    return gistInfo.files[filename].raw_url;
+  } catch (e) {
+    return null;
```

Comment:
I opted to keep this code in the FFI rather than implement the decoding in PureScript, once again to keep the diff minimal. In the future, though, it would be nice to switch to something like `codec-argonaut` instead.